apps/maestro/src/ui/pages/InterchainTokenDetailsPage/TokenDetailsSection.tsx

```diff
@@ -221,6 +221,8 @@ const TokenDetailsSection: FC<TokenDetailsSectionProps> = (props) => {
   function getTokenExplorerLink() {
     if (isSuiChain) {
       return `${props.chain.explorer.url}/object/${props.tokenAddress}`;
+    } else if (props.chain.chain_name.includes("stellar")) {
```

Comment:
use `.id` here too and refactor `isSuiChain` to do the same thing for consistency. it's currently using `chain_id` (number).